.github/rever.xsh

```diff
@@ -1,7 +1,7 @@
 $ACTIVITIES = ["authors", "changelog"]
 
 # Basic settings
-$PROJECT = $GITHUB_REPO = "conda"
+$PROJECT = $GITHUB_REPO = @($(basename $(git remote get-url origin)).split('.')[0])
```

Comment:
What's the @ in front?

Reply:
it wasn't necessary after all but this is where it comes from: https://xon.sh/tutorial.html?highlight=script#python-evaluation-with

Reply:
Nice, gotcha, thanks!